include/apps/tools/OswAppWebserver.h

```diff
@@ -17,5 +17,6 @@ class OswAppWebserver : public OswApp {
 
   private:
     OswUI* ui;
+    void WifiConfigScreen();
```

Comment:
how about `connectionInfo()`?

Reply:
ok It can be renamed no problem. This change was made to clean up this section with future code work in mind.

Reply:
WifiConfigScreen renamed to ConnectionInfo